### PR TITLE
AEA-2098 Compare Prescription EPSAT Updates

### PR DESCRIPTION
**Pull request by booshibooshi**

### CODE REVIEW

File: tool/site/client/src/components/my-prescriptions/prescriptionGroupTable.tsx [hunk 1]
```diff
@@ -40,9 +44,26 @@ export const PrescriptionGroupTable: React.FC<PrescriptionGroupTableProps> = ({
         <Table.Body>
           {prescriptions.map((prescription, index) =>
             <Table.Row key={index}>
-              <Table.Cell>{prescription}</Table.Cell>
               <Table.Cell>
-                <PrescriptionActions prescriptionId={prescription} {...actions}/>
+                <Label>{prescription}</Label>
+                <Checkboxes id={`prescription.${prescription}`}>
+                  <Checkboxes.Box
```
Comment (booshibooshi) [inline tool/site/client/src/components/my-prescriptions/prescriptionGroupTable.tsx:50]: move to component

File: tool/site/client/src/pages/comparePage.tsx [hunk 2]
```diff
@@ -1,79 +1,77 @@
...
+              <ReactDiffViewer
+                oldValue={compareResult.prescription1}
+                newValue={compareResult.prescription2}
+                splitView={true}
+                compareMethod={DiffMethod.WORDS}
+              />
+            </>
+            : <Formik<ComparePrescriptions>
+              initialValues={initialValues}
+              onSubmit={null/*setComparePrescriptions*/}
```
Comment (booshibooshi) [inline tool/site/client/src/pages/comparePage.tsx:38]: to handle custom input again

File: tool/site/server/src/routes/state/session.ts [hunk 3]
```diff
@@ -41,5 +41,89 @@ export default [
       )
       return h.response(bundles).code(200)
     }
+  },
+  {
+    method: "GET",
+    path: "/api/compare-prescriptions",
+    handler: async (request: Hapi.Request, h: Hapi.ResponseToolkit): Promise<Hapi.ResponseObject> => {
```
Comment (booshibooshi) [inline tool/site/server/src/routes/state/session.ts:48]: to move and re-categorize routes

File: tool/site/client/src/components/my-prescriptions/prescriptionGroupTable.tsx [hunk 4]
```diff
@@ -51,3 +72,21 @@ export const PrescriptionGroupTable: React.FC<PrescriptionGroupTableProps> = ({
...
+) {
+  const addToCompare = ((event.target) as HTMLInputElement).checked
+  const removeFromCompare = !addToCompare
+  if (addToCompare) {
+    const comparePrescriptions = (await axiosInstance.post(`${baseUrl}api/compare-prescriptions`, {name: name.toLowerCase().replace(" ", "_"), id})).data
+    if (comparePrescriptions.prescription1 && comparePrescriptions.prescription2) {
+      redirect(`${baseUrl}compare-prescriptions`)
+    }
+  } else if (removeFromCompare) {
+    await axiosInstance.post(`${baseUrl}api/compare-prescriptions`, {name: "", id: ""})
```
Comment (booshibooshi) [inline tool/site/client/src/components/my-prescriptions/prescriptionGroupTable.tsx:90]: to handle removing correct prescription

File: tool/site/client/src/fhir/helpers.ts [hunk 5]
```diff
@@ -120,3 +120,41 @@ export function isRepeatDispensing(bundle: fhir.Bundle): boolean {
     .flatMap(medicationRequest => medicationRequest.courseOfTherapyType.coding)
     .some(coding => coding.code === "continuous-repeat-dispensing")
 }
+
+export function orderBundleResources(r1: fhir.Resource, r2: fhir.Resource): number {
+  return getSortIndex(r1) - getSortIndex(r2)
+}
+
+function getSortIndex(resource: fhir.Resource) {
```
Comment (callumtipper-IW) [inline tool/site/client/src/fhir/helpers.ts:128]: return straight from the case?

**Review by callumtipper-IW — APPROVED** [review]

**Pull request merged**